src/nvim/tui/tui.c

```diff
@@ -1415,7 +1414,7 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
     } else if (konsole || xterm || gnome || rxvt || st || putty
         || linuxvt  // Linux 4.8+ supports 256-colour SGR.
         || mate_pretending_xterm || gnome_pretending_xterm
-        || tmux || tmux_pretending_screen
```

Comment:
because `tmux_pretending_screen` relied on the `os_getenv` "hack" to set it. Now that check in included in the tmux bool instead.